rewards/df-intro.md

```diff
@@ -6,50 +6,95 @@ description: Learn the fundamentals of The Way of Data Farming 🧑‍🏫
 | 50%             | 50%            |
 
-#### What are Passive Rewards?
 
-Passive rewards are the OCEAN rewards paid to Data Farmers just for locking their OCEAN tokens.
+<figure><img src="../.gitbook/assets/rewards/df-reward-streams.png" alt=""><figcaption><p>Rewards Page overview for each stream.</p></figcaption></figure>
+
+### What are Passive Rewards?
+
+Passive rewards are the OCEAN rewards paid to Data Farmers just for locking their OCEAN tokens.  
+
+veOCEAN holders can generate yield completely passively if they wish, though they are incentivized with larger real yield if they **actively participate** in farming yield from assets, or completing predictionc hallenges.  
 
 <figure><img src="../.gitbook/assets/gif/passive-income.gif" alt=""><figcaption></figcaption></figure>
 
-[To start getting passive rewards, go here.](../user-guides/get-started-df.md)
+[To start getting passive rewards, go here.](../user-guides/data-farming/how-to-veocean.md)
+
+### What are Active Rewards?
 
-#### What are Active Rewards?
+Active Rewards are OCEAN rewards paid to contributors actively working to improve Ocean Protocol.  
 
-Active rewards are OCEAN rewards paid to Data Farmers that allocate their veOCEAN tokens to Ocean ecosystem assets. They're called Active rewards because the amount of rewards relies on the active participation of the Data Farmer to select and allocate veOCEAN to these assets. **Active rewards yield depends on the sales of allocated assets.** No sales = no rewards, so choose your favorites wisely & then allocate. Always DYOR.
+**Active Rewards** is then composed of two sub-streams:
+1. 5,000 OCEAN is paid out to Challenge DF.
+2. 70,000 OCEAN remainder is paid out to Volume DF.  
 
-Active rewards are governed and defined by the [Reward Function](df-max-out-yield.md#reward-schedule).
+| Challenge DF | Volume DF |
+| --------------- | -------------- |
+| 5,000 OCEAN     | 70,000 OCEAN    |
 
-[To start getting active rewards, go here.](../user-guides/how-to-data-farm.md)
+### What is Volume DF
+
+[Volume DF](df-volumedf.md#allocating-veocean) rewards Data Farmers that allocate their veOCEAN tokens to Ocean datasets. It's called Volume DF because the amount of rewards relies on assets that are actively generating Data Consume Volume.  
+
+You can visit the follow tutorial learn how to [access Volume DF](../user-guides/data-farming/how-to-volumedf.md).  
+
+Volume DF rewards are governed and defined by the [Reward Function](df-intro.md#reward-schedule).  
 
 #### Estimating APY
 
-To estimate your APY effectively, you will need a good understanding of how Passive and Active Rewards work. Therefore you should make sure to complete reading the documentation on Data Farming.
+To a get a good sense for how much APY you'll make, we recommend you understand the details of [how veOCEAN works](df-veocean.md). This will give you a basis for the calculations and provide you with an outline for what to expect in terms of rewards over time.  
```

Comment:
I assume is: To get a good sense. Extra "a" in between to and get

Reply:
fixed